Source/help.cpp

```diff
@@ -384,13 +384,19 @@ const char gszHelpText[] = {
 	"C: Open Character screen|"
 	"Q: Open Quest log|"
 	"F: Reduce screen brightness|"
-	"G: Increase screen brightness|"
+	"G: Increase screen brightness|"
+	"W: Switch weapon sets|"
 	"Z: Zoom Game Screen|"
 	"+ / -: Zoom Automap|"
 	"1 - 8: Use Belt item|"
+<<<<<<< HEAD
+	"F5, F6, F7, F8:     Set hot key for skill or spell|"
+	"Shift + Left Click: Attack without moving|"
+=======
 	"F5, F6, F7, F8:     Set hotkey for skill or spell|"
 	"Shift + Left Click: Attack without moving|"
 	"Shift + Left Click (on character screen): Assign all stat points|"
+>>>>>>> f49b55cbe1e56744faf616cdde64b3a8202b1024

```

Comment:
Conflict sad :(

Reply:
No idea what just happened... it's definitely a git user error on my side though. 
I'll take it as a learning experience but not sure I can recover this... May have to start with a fresh branch.

Reply:
Just edit the file, correct the duplicate lines, commit and put.

Reply:
Thanks - Done that and it seems to build. Let's see if it runs :)